tests/scripts/all.sh

```diff
@@ -726,6 +726,7 @@ component_test_no_platform () {
     scripts/config.pl unset MBEDTLS_FS_IO
     scripts/config.pl unset MBEDTLS_PSA_CRYPTO_STORAGE_C
     scripts/config.pl unset MBEDTLS_PSA_ITS_FILE_C
+    scripts/config.pl unset MBEDTLS_PSA_ITS_FILE_C_INCLUDE_TEST
```

Comment:
We shouldn't need to add these lines when the `MBEDTLS_PSA_ITS_FILE_C_INCLUDE_TEST` option is removed.

Reply:
Done